plugins/woocommerce/src/Internal/Admin/Agentic/AgenticController.php

```diff
@@ -29,7 +27,16 @@ public function register() {
 			return;
 		}
 
-		// Only initialize if the agentic checkout feature is enabled.
+		add_action( 'init', array( $this, 'on_init' ) );
```

Comment:
```suggestion
		add_action( 'woocommerce_init', array( $this, 'on_init' ) );
```

Minor but `woocommerce_init` is also called on `init`, but at that stage, Woo core should already be initialized. I think that makes the hook more suitable?

@htdat I want to merge this PR quickly. What do you think about this? If you agree, I'll apply the change and merge.

Reply:
Yup, `woocommerce_init` is even better. Let's merge this one!

Reply:
With `woocommerce_init`? The answer is a bit ambiguous :D